social_core/backends/apple.py

```diff
@@ -128,8 +128,8 @@ def decode_id_token(self, id_token):
                 audience=self.get_audience(),
                 algorithms=['RS256'],
             )
-        except PyJWTError:
-            raise AuthFailed(self, 'Token validation failed')
+        except PyJWTError as error:
+            raise AuthFailed(self, 'Token validation failed by: ' + error.__str__())
```

Comment:
Using f-string will make the code more readable:

```suggestion
            raise AuthFailed(self, f'Token validation failed by {error}')
```

Still, I'm not convinced whether it's desired to expose this to the user. Won't PyJWT leak credentials in the exceptions?

Reply:
@nijel thanks for revision. In my test the credentials don't leak.  Only the cause of the error is printed. I use social-core integred with [drf-social-oauth](https://github.com/wagnerdelima/drf-social-oauth2) in case you're wondering why the REST call in the log

![image](https://user-images.githubusercontent.com/5522872/143960059-ce0bb632-0d23-42ea-a68e-d252c38aa78e.png)

PD: Sorry for my basic english